home/tests/test_models.py

```diff
@@ -1043,11 +1054,6 @@ def test_template_create_with_buttons(self):
             locale=Locale.objects.get(language_code="en"),
         )
         wat.save()
-        created_qr, _ = TemplateContentQuickReply.objects.get_or_create(name="Button1")
-        wat.quick_replies.add(created_qr)
-        created_qr, _ = TemplateContentQuickReply.objects.get_or_create(name="Button2")
-        wat.quick_replies.add(created_qr)
-        wat.save()
```

Comment:
I think we'll need to add these into the `WhatsAppTemplate` as buttons, otherwise the `test_template_create_with_buttons` test name no longer makes sense..